### ISSUE TITLE
The model not trained on all of my classes

### ISSUE THREAD

**Issue by PAIN-BARHAM** [ultralytics/ultralytics#1651]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

Not all of my classes are shown on the Confusion Matrix nor on the model summary after finishing the training.
from ultralytics import YOLO

Model Loading: 

```
# Load a model
#model = YOLO('yolov8n.yaml')  # build a new model from YAML
model = YOLO('yolov8n.pt')  # load a pretrained model (recommended for training)
#model = YOLO('yolov8n.yaml').load('yolov8n.pt')  # build from YAML and transfer weights

# Train the model
model.train(data='/home/think3/Desktop/test_yolo/datasets/dataset/data.yaml', epochs=20, imgsz=1280, batch=-1, plots=True)
```
Model Summary: 

```
Validating runs/detect/train21/weights/best.pt...
Ultralytics YOLOv8.0.20 🚀 Python-3.8.10 torch-1.13.1+cu117 CUDA:0 (Tesla V100-PCIE-32GB, 32511MiB)
Model summary (fused): 168 layers, 3011693 parameters, 0 gradients, 8.1 GFLOPs
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 16/16 [00:47<00:00,  2.95s/it]
                   all       1506       1506      0.162      0.146      0.163      0.142
                   Ain       1506        248          0          0          0          0
                    Al       1506        264          0          0          0          0
                  Alef       1506        280          0          0          0          0
                   Beh       1506        299          0          0          0          0
                   Dad       1506        259          0          0          0          0
                   Dal       1506        156      0.972      0.878       0.98      0.852
Speed: 1.4ms pre-process, 10.8ms inference, 0.0ms loss, 1.1ms post-process per image
Results saved to [1mruns/detect/train21[0m
```

### Environment

- Ultralytics YOLOv8.0.20 🚀 Python-3.8.10 torch-1.13.1+cu117 CUDA:0 (Tesla V100-PCIE-32GB, 32511MiB)
- OS: Ubuntu 20.04

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@PAIN-BARHAM removing bug label.

Your model is trained on all data you've supplied. If you have not supplied val labels for certain classes then what would you expect us to show for those classes?
